tracker/urls.py

```diff
@@ -29,6 +31,8 @@
     path('health/ok/', common_views.health_ok),
     path('health/version/', common_views.health_version),
     path('csrf/', common_views.get_csrf_token, name='csrf_token'),
+    path('sitemap.xml', sitemap),
```

Comment:
I wonder if we need to give this view some special treatment with regards to the cloudflare cache.

Reply:
You mean cache bursting when a new page is added? That's a good question. I am not sure how we would do that without messing with the sitemap internals of wagtail.

Reply:
Yes. Honestly I was thinking that this page should ideally be cached (I am not fully sure how adverse it is performance wise). Else the bots checking sitemap link shouldn't bring down our website. But I know cache bursting will be an issue as well. So I am not sure either. @harrislapiroff thoughts?

Reply:
Definitely should be cached. Can we register a signal to bust the cache for this path whenever a page is published? Otherwise we could just set the TTL shorter for this endpoint.